opencti-platform/opencti-front/tests_e2e/report/deleteReport.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, test } from "../fixtures/baseFixtures";
+import { ReportPage } from "../model/report.pageModel";
+import { ReportDetailsPage } from "../model/reportDetails.pageModel";
+
+test.skip('Create a new report page and delete it', async ({ page }) => {
+  const reportPage = new ReportPage(page);
+  const reportDetailsPage = new ReportDetailsPage(page);
+  await page.goto('/dashboard/analyses/reports');
+  await reportPage.addNewReport();
+  await reportPage.getReportNameInput().click();
+  await reportPage.getReportNameInput().fill('Test delete report e2e');
+  await reportPage.getCreateReportButton().click();
+  await page.waitForTimeout(1000);
```

Comment:
That's strange we need that as the standard behavior of playright is by default to wait until it can do the action, and in the config we wrote to wait 20 seconds so it should be ok.

My first impression is because we need to await both the click AND the getCreateButton. I'll do some tests on my side locally.